ffi-convert/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ffi-convert"
-version = "0.5.0-pre"
+version = "0.6.0-pre"
```

Comment:
no need to update this, as the 0.5.0 has not been released yet (as denoted by the `pre` for "preversion")